modules/backend/widgets/Filter.php

```diff
@@ -566,7 +566,11 @@ protected function makeFilterScope($name, $config)
         /*
          * Set scope value
          */
-        $scope->value = $this->getScopeValue($scope);
+        if($scopeType=='checkbox' || $scopeType=='switch'){
+            $scope->value = isset($config['value']) ? $config['value'] : $this->getScopeValue($scope);
```

Comment:
The config option should be `'default'`, not `'value'`. Also, does this prevent the user from overriding the default? It seems like if a default is defined then the value retrieved from `$this->getScopeValue($scope)` would not be used.

Reply:
@LukeTowers  done